omnisafe/common/buffer/vector_offpolicy_buffer.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2022 OmniSafe Team. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Implementation of ReplayBuffer."""
+
+from typing import Dict
+
+import torch
+from gymnasium.spaces import Box
+
+from omnisafe.common.buffer.offpolicy_buffer import OffPolicyBuffer
+from omnisafe.typing import OmnisafeSpace
+
+
+class VectorOffPolicyBuffer(OffPolicyBuffer):
+    """A ReplayBuffer for off_policy Algorithms."""
+
+    def __init__(  # pylint: disable=super-init-not-called,  too-many-arguments
+        self,
+        obs_space: OmnisafeSpace,
+        act_space: OmnisafeSpace,
+        size: int,
+        batch_size: int,
+        num_envs: int,
+        device: torch.device = torch.device('cpu'),
+    ):
```

Comment:
need docstring

Reply:
docstring already defined in the BaseBuffer(abstract class)